js/build.js

```diff
@@ -91,6 +97,15 @@ Fliplet.Widget.instance('image', function (data) {
     return;
   }
 
+  // Check when image is loaded
+  $('[data-image-id="' + data.id + '"]').each(function(index, img) {
```

Comment:
this is already running in a `each` cycle for each image, see line 1:

```
Fliplet.Widget.instance('image', function (data) {
```

Reply:
@squallstar Thanks! Tweaked with latest commit.